uni/test/test_widget.dart

```diff
@@ -11,6 +12,12 @@ Widget testableWidget(Widget widget,
 Widget wrapWidget(Widget widget) {
   return MaterialApp(
       home: Scaffold(
-    body: widget,
+    body: Localizations(
+      delegates: const [
+        S.delegate,
+      ],
+      locale: const Locale('pt'),
+      child: widget,
+    ),
```

Comment:
Remove Localizations widget and add to this MaterialApp the properties you already added in the MaterialApp of the app, like **locale**, **localizationsDelegates** and **supportedLocales**.
This will fixs half the tests issues